overlord/fdestate/backend/reseal.go

```diff
@@ -105,12 +105,12 @@ func getUniqueModels(bootChains []boot.BootChain) []secboot.ModelForSealing {
 }
 
 func doReseal(manager FDEStateManager, method device.SealingMethod, rootdir string) error {
-	_, runModels, runPCRProfile, err := manager.Get("run+recover", "all")
+	hasRunParameters, _, runModels, runPCRProfile, err := manager.Get("run+recover", "all")
 	if err != nil {
 		return err
 	}
 
-	_, recoveryModels, recoveryPCRProfile, err := manager.Get("recover", "system-save")
+	hasRecoverParameters, _, recoveryModels, recoveryPCRProfile, err := manager.Get("recover", "system-save")
```

Comment:
is it actually possible in practice to have one set of parameters but not the other?

Reply:
I suppose if we change the kernel command line for example, the recovery would not change. But the run would.